wavefront_sdk/common/heartbeater_service.py

```diff
@@ -41,15 +41,16 @@ def __init__(self, wavefront_client, application_tags, components, source):
         if isinstance(components, str):
             components = [components]
         for component in components:
-            self.heartbeat_metric_tags_list.append(
-                {
+            metric_tags = {
                     APPLICATION_TAG_KEY: application_tags.application,
```

Comment:
Do we always expect `application_tags` to be not `None`

Reply:
yep